.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Continuous Integration
```

Comment:
This is going to be a bit wordy at the beginning of each check. I tried a few weeks ago to use an empty name to get rid of the leading "$name / " in check names but that didn't work, then. I'd just pick something short, but it's up to you.